src/FilterList.js

```diff
@@ -39,11 +40,43 @@ export let Label = inject(_.pick('tree'))(
                 style={{
                   userSelect: 'none',
                   marginTop: '0.5rem',
-                  width: '5.5rem',
                   transform: 'translateX(-2.25rem)',
                   lineHeight: '1.4rem',
                 }}
               >
+                <Item
+                  style={{
+                    fontWeight: 'bold',
+                    color: 'initial',
+                    cursor: 'initial',
+                  }}
+                >
+                  {F.autoLabel(node.type)}
+                </Item>
+                {_.map(
+                  x => (
+                    <Item
+                      key={x.value}
+                      onClick={() =>
+                        tree.replace(
+                          node.path,
+                          newNodeFromType(x.value, fields, node)
+                        )
+                      }
+                    >
+                      —Change to {x.label}
+                    </Item>
+                  ),
+                  F.autoLabelOptions(
+                    _.without(
+                      [node.type],
+                      _.get([node.field, 'typeOptions'], fields)
+                    ) || []
+                  )
+                )}
+                <div
+                  style={{ borderBottom: '1px solid #eee', margin: '4px -5px' }}
```

Comment:
This probably shouldn't be in the core components and should be a themeable thing, e.g. something like `className="dropdown-option-separator"` or whatever and then themed in grey vest